api/services/Slack.js

```diff
@@ -0,0 +1,34 @@
+var format = require('util').format
+var Promise = require('bluebird')
+var request = require('request')
+var post = Promise.promisify(request.post)
```

Comment:
haha, actually I had ben doing it that way, but Lawrence mentioned he'd been doing it this way in other places in the app, so I figured I'd just keep on that way :) cheers though Harlan. Keen for other pointers whenever you have them